octoprint_mrbeam/__init__.py

```diff
@@ -1748,162 +1748,174 @@ def gcodeConvertCommand(self):
         del data["gcodeFilesToAppend"]
 
         if command == "convert":
```

Comment:
Shouln't the content of this `if` statement be isolated with the rest of the slicer files?